crates/router/src/core/admin.rs

```diff
@@ -438,19 +424,15 @@ pub async fn retrieve_payment_connector(
     let _merchant_account = store
         .find_merchant_account_by_merchant_id(&merchant_id)
         .await
-        .map_err(|error| {
-            error.to_not_found_response(errors::ApiErrorResponse::MerchantAccountNotFound)
-        })?;
+        .to_not_found_response(errors::ApiErrorResponse::MerchantAccountNotFound)?;
 
     let mca = store
         .find_by_merchant_connector_account_merchant_id_merchant_connector_id(
             &merchant_id,
             &merchant_connector_id,
         )
         .await
-        .map_err(|error| {
-            error.to_not_found_response(errors::ApiErrorResponse::MerchantConnectorAccountNotFound)
-        })?;
+        .to_not_found_response(errors::ApiErrorResponse::MerchantConnectorAccountNotFound)?;
```

Comment:
Same here, would be better if we add the `connector_label` field

Reply:
Not urgent, but good to have. Yes better to create the issue. Please create one with the good first issue label and mention it here

Reply:
created an issue here #899